src/Aspire.Hosting.SqlServer/Aspire.Hosting.SqlServer.csproj

```diff
@@ -14,6 +14,8 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Aspire.Hosting\Aspire.Hosting.csproj" />
+
+    <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
```

Comment:
Can we just compile the file into the tests instead? I know other `Aspire.Hosting.X` assemblies do this, but they need it for other reasons and I'd like to limit when we use IVT.